.github/workflows/pypi_publish.yml

```
@@ -70,14 +77,20 @@ jobs:
       - name: Set Version (workflow_dispatch)
         if: github.event_name == 'workflow_dispatch'
         run: |
-          echo "Version set to ${{ github.event.inputs.version }}"
+          echo "Version input set to ${{ github.event.inputs.version }}"
+          # Remove the 'v' prefix if it exists
+          VERSION=${VERSION#v}
           echo "VERSION=${{ github.event.inputs.version }}" >> $GITHUB_ENV
 
       - name: Set Version (tag)
         if: startsWith(github.ref, 'refs/tags/v')
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:6:30: Double quote to prevent globbing and word splitting

(shellcheck)